### DISCUSSION TITLE
Support using/hiding .github templates while providing additional repo templates

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#16383] [Projects and Issues]
GitHub supports defining organization level default issue templates, it's kinda documented here:
* [Configuring issue templates for your repository](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
* [Community health file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)

I've run into an organization that wants to unify most of their issue templates by using their `.github` repository to define a number of issue templates. But, they'd be interested in adding a couple of additional repository specific issue templates.

What they don't want to do is have two copies of their main issue templates (one in their `.github` repository and their specific repository).

Offhand, I'd like to see this as a config file that allows a repository to opt into keeping a certain named set of templates from the `.github` repository. Any template not listed would be clipped.
Or vice versa, the ability to hide a certain named set of templates from the `.github` repository. Any template not excluded by the config would be included.

The presence of this config file would result in any additional local templates being used as supplemental templates to the ones from the `.github` repository instead of suppressing all of the templates from that repository.

**Comment by cjbarth** [comment]
Even better would be allowing choosing which template to use for a given issue type. In stead of using fixed files names, let the configuration screen let us choose which file to use. We can then pick across repos in the same organization. I should be able to specify in the `config.yml` of a repo, which template I'd like to use.

It does seem that this might be possible using `config.yml`, but there is virtually no documentation (that I can find) on setting up a `config.yml` outside of this [sample](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms).

**Comment by gabrielbdornas** [comment]
Hello @jsoref, nice suggestion. We also need this kind of feature to enhance our organization's management.

See https://github.com/splor-mg/.github/issues/2
